src/components/Textarea/index.tsx

```diff
@@ -0,0 +1,102 @@
+import type { CSSProperties, HTMLAttributes, ReactElement } from 'react'
+import { useCallback, useEffect, useRef } from 'react'
+import styled from '@emotion/styled'
+
+export interface TextAreaProps extends HTMLAttributes<HTMLDivElement> {
+  label?: string
+  placeholder?: string
+  children: string
+  guideMessage?: string
+  textAreaStyle?: CSSProperties
+  BgType: 'filled' | 'ghost'
```

Comment:
```suggestion
  bgType: 'filled' | 'ghost'
```
camel case로 통일하는게 좋을 것 같습니당!